data/experiment-recipe-samples/mobile-a-a.json

```diff
@@ -1,4 +1,5 @@
 {
+  "schemaVersion": "0.1.0",
```

Comment:
I honestly can't imagine us using the patch version for anything useful, but I also have no objection to having it here, so 👍